tests/integration/test_charm.py

```diff
@@ -46,6 +46,7 @@ async def test_cos_agent_relation(app: Application):
         "grafana-agent",
         application_name=f"grafana-agent-{app.name}",
         channel="latest/edge",
+        series="jammy",
```

Comment:
required, otherwise it tries to deploy on noble which is not supported on juju 3.2 , and the charm currently also only builds on top of jammy , see also https://matrix.to/#/!UPTldjWgQImSaCAjYf:ubuntu.com/$voOAgZSbo-Jg-yyA-If33PlXutLyaGGrFbQg6FOD4D0?via=ubuntu.com&via=matrix.org